core/api/transport/impl/http/http_listener_impl.hpp

```diff
@@ -38,11 +38,11 @@ namespace kagome::api {
      * @param endpoint loopback ip address to listen
      * @param http_config http session configuration
      */
-    ListenerImpl(Context &context,
-                 const Configuration &configuration,
-                 HttpSession::Configuration http_config);
+    HttpListenerImpl(Context &context,
+					 const Configuration &configuration,
```

Comment:
Are these tabs or it's just github viewer?

Reply:
applied clang-format